README.md

```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Surver Form
+> This is a project that shows my code review skills using Github Review features
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  A browser
+```
+### Setup
+Clone the repository and change the directory to past-project
+``` 
+  $ git clone git@github.com:Felix45/review-past-project.git
+
+  $ cd past-project
+
+```
+
+### Usage
+Start the http application server
+``` 
+  Open with live server
+```
+
+### Deployment
+- [Visit Application Homepage](http://localhost:8080)
+
+
+### Authors
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Felix Ouma|[@Felix45](https://github.com/Felix45)|[@Felix_Atonoh](https://twitter.com/Felix_Atonoh)|[LinkedIn](https://www.linkedin.com/in/felix-ouma-639766b0/)|
+
+
+
```

Comment:
Please consider adding an Acknowledgment section to your readme file, acknowledging [Microverse](https://microverse.org) as the organization that gave you this project to work on. Giving attribution to shared works shows a person how professional you are.